### PR TITLE
[Data objects] Geo data fields are not scalable

### LINKED ISSUE #6152: [Data objects] Geo data fields are not scalable
# Feature Request
## Feature description
The GEO data fields cannot be adjusted in height or width. This results in an inconsistent width in many object views.

### PR DESCRIPTION
**Pull request by skoch98**
## Changes in this pull request  
Resolves #6152

### CODE REVIEW

**Comment by brusch** [comment]
@skoch98 @skochTC  can you please resolve the conflicts? Thanks a lot!

File: lib/Bootstrap.php [hunk 1]
```diff
@@ -125,6 +125,8 @@ public static function bootstrap()
             $loader = include __DIR__ . '/../../../../vendor/autoload.php';
         } elseif (getenv('PIMCORE_PROJECT_ROOT') != '' && file_exists(getenv('PIMCORE_PROJECT_ROOT') . '/vendor/autoload.php')) {
             $loader = include getenv('PIMCORE_PROJECT_ROOT') . '/vendor/autoload.php';
+        } elseif (defined('PIMCORE_PROJECT_ROOT') && file_exists(PIMCORE_PROJECT_ROOT . '/vendor/autoload.php')) {
```
Comment (brusch) [inline lib/Bootstrap.php:128]: This doesn't seem to be related, can you please revert this change?

**Review by brusch — CHANGES_REQUESTED** [review]

File: lib/Bootstrap.php (hunk 1 above)
Comment (skochTC) [reply lib/Bootstrap.php]: Oops, I think I just made it worser.

I simply copied the file from the master.

I'll look at it again after the break. Sorry!

Comment (skochTC) [reply lib/Bootstrap.php]: I don't know why the file is still shown under changes, it is identical to the file from the master.

I'm checking the merge confliction now.

**Comment by skochTC** [comment]
> 
> 
> @skoch98 @skochTC can you please resolve the conflicts? Thanks a lot!

Some strange commits because of the Bootstrap.php but now everything is done.
